Minio.Functional.Tests/Program.cs

```diff
@@ -68,7 +66,7 @@ public static void Main(string[] args)
                 .WithHttpClient(httpClient)
                 .Build();
         else
-            minioClient = new MinioClient()
+            minioClient = new MinioClient(httpClient)
                 .WithCredentials(accessKey, secretKey)
```

Comment:
You should use WithHttpClient to follow builder pattern.

Reply:
Ok

Reply:
Done.